server/master/src/main/java/org/apache/accumulo/master/MasterClientServiceHandler.java

```diff
@@ -253,6 +253,16 @@ public void waitForFlush(TInfo tinfo, TCredentials c, String tableId, ByteBuffer
 
   }
 
+  private String getNamespaceId(TableOperation tableOp, String tableId) throws ThriftTableOperationException {
```

Comment:
For consistency, could use the method name`getNamespaceIdFromTableId` like you did elsewhere.